lib/vtls/schannel.c

```diff
@@ -335,6 +335,11 @@ set_ssl_ciphers(SCHANNEL_CRED *schannel_cred, char *ciphers)
       alg = get_alg_id_by_name(startCur);
     if(alg)
       algIds[algCount++] = alg;
+#ifdef SCH_USE_STRONG_CRYPTO
```

Comment:
Checks revealed that old toolchains don't provide `SCH_USE_STRONG_CRYPTO`. Is it better to do this, or should I rather do
```
#ifndef SCH_USE_STRONG_CRYPTO
#define SCH_USE_STRONG_CRYPTO             0x00400000
#endif
```

Reply:
> should I rather do
> 
> ```
> #ifndef SCH_USE_STRONG_CRYPTO
> #define SCH_USE_STRONG_CRYPTO             0x00400000
> #endif
> ```

yes